server.js

```diff
@@ -30,12 +32,69 @@ io.of(/^\/.*$/).on("connection", (socket) => {
   });
 
   socket.on("update", (msg) => {
+    console.log("update", msg);
     Object.assign(state[nsp], msg);
     socket.nsp.emit("state", state[nsp]);
   });
 });
 
-app.use(express.static("public"));
+// Proxy to Daily API to create rooms for breakout rooms
+app.post("/create-rooms", async (req, res) => {
+  // Add logic to check if the user is authorized to create a room
+  const isAllowed = true;
```

Comment:
We could probably use `isOwner` from a meeting token here somehow, but that will be part of a larger conversation about this API proxy thing. I'm fine shipping this as-is :)

Reply:
I called this out on the actual docs page as a point of inspiration. I believe @harshithpabbati did this on his demo, it gets a bit tricky with some edge cases but it's a great feature to point customers to if they want more info